### ISSUE TITLE
Rich text editing - Create / finalize Figma template

### ISSUE THREAD

**Issue by halocline** [grommet/hpe-design-system#3287]
Parent issue: #3331 

Create, publish, and announce a Figma template which can be added to designs in Figma.

Dependent on #3222 and findings/refinements from #3332

**Comment by halocline** [comment]
Won't do. Complexity doesn't justify our efforts at this time. This feature was needed for an internal (i.e. not customer facing) use case.

Instead the recommendation is to author these posts in markdown and use the Markdown component.

If a true text editor is need, the current recommendation is to look at [QuillJS](https://github.com/slab/quill) because of active development, accessibility, and adoption by large companies like Salesforce and Slack.
